src/slurm_cloud_bursting/resume.py

```diff
@@ -0,0 +1,262 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with
+# the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+import collections
+import logging
+import re
+import subprocess
+from logging.config import fileConfig
+
+import argparse
+import boto3
+from botocore.config import Config
+from botocore.exceptions import ClientError
+from configparser import ConfigParser
+from retrying import retry
+
+from common.schedulers.slurm_commands import get_nodes_info, set_nodes_down, set_nodes_power_down, update_nodes
+from common.utils import grouper
+from slurm_cloud_bursting.utils import CONFIG_FILE_PATH
+
+TAG_SPECIFICATIONS = [
+    {
+        "ResourceType": "instance",
+        "Tags": [{"Key": "aws-parallelcluster-slurm-node-type", "Value": "cloud-bursting-compute"}],
+    }
+]
+EC2Instance = collections.namedtuple("EC2Instance", ["id", "private_ip", "hostname"])
+LOG_CONFIG_FILE = "/opt/parallelcluster/configs/slurm/parallelcluster_resume_logging.conf"
```

Comment:
as discussed this probably should be in the config file and in case not found default to a default logging config that needs to be shipped with the code itself.

Reply:
Done